henry/cli.py

```diff
@@ -226,7 +226,8 @@ def add_common_arguments(parser: argparse.ArgumentParser):
         "--save", action="store_true", default=False, help="Save output to CSV.",
     )
     parser.add_argument("-q", "--quiet", action="store_true", help="Silence output")
-    parser.add_argument("--timeout", type=int, default=120, help=argparse.SUPPRESS)
+    parser.add_argument("--timeout", type=int, default=120,
```

Comment:
auto formatting

Reply:
what did you use to format? This project uses `black`, make sure  your IDE is not overriding that.